### PR TITLE
pyextract: add extract coredump support

### PR DESCRIPTION
**Pull request by Junbo-Zheng**
pyextract: add extract coredump support

Signed-off-by: Junbo Zheng <zhengjunbo1@xiaomi.com>

### CODE REVIEW

**Comment by Junbo-Zheng** [comment]
I have added the function of extracting `core dump` files. 

A core file will be generated if the `core dump` configuration is turned on. The `core dump` will also be extracted to the current path for easy debugging when extracting and merging logs.

No other operations are required, the `core dump` file will be extracted by default, and the usage is no different from before.

**Comment by Junbo-Zheng** [comment]
@nowhszh PTAL

File: pyextract.py [hunk 1]
```diff
@@ -250,6 +259,18 @@ def __gunzip_all__(self):
...
 
+    def extract_coredump(self):
+        coredump = self.__find_coredump()
+        if len(coredump) == 0:
+            print("Coredump not found")
+            return 0
+
+        print("Coredump found")
+        des_path = os.path.join(os.getcwd())
+        copy_cmd = "cp " + coredump[0] + " " + des_path
```
Comment (nowhszh) [inline pyextract.py]: why extract array coredump index 1 only?

Comment (Junbo-Zheng) [reply pyextract.py]: > why extract array coredump index 1 only?

@nowhszh  It's a nice catch, addressed, please review again.

**Review by nowhszh — APPROVED** [review]
LGTM

**Pull request merged**